docs/goals-canvas.md.txt

```diff
@@ -0,0 +1,26 @@
+| Zoek of raadpleeg de leeftijd van persoon | Door te zoeken of raadplegen en/of te filteren op het kenmerk leeftijd | bovenstaande zoek en raadpleegopties + een naamfilter waarbij leeftijd wordt opgegeven | (gefilterde) persoon bevat naast bovenstaande gegevens ook leeftijd | Bepalen of persoon aan criteria voldoet, zonder daarbij onnodig persoonsgegevens te raadplegen. | [774](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/736){:target="_blank"}, [691](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/691){:target="_blank"}, [689](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/689){:target="_blank"}, [690](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/691){:target="_blank"}, [772](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/772){:target="_blank"}, [773](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/737){:target="_blank"}, |
```

Comment:
Net als hierboven zou ik bij what 'zoek of raadpleeg' weglaten.
Bij how zou ik zeggen: bij gebruik filter, moet 'leeftijd' worden toegevoegd.
input: default: niets, bij filter: leeftijd en/of [relatie].leeftijd waarbij [relatie] = kind, ouder, partner